joystick_interrupt/src/joystick_interrupt.cpp

```diff
@@ -70,6 +79,10 @@ class JoystickInterrupt
     }
     if (!msg->buttons[interrupt_button_])
     {
+      if (last_joy_msg_ != ros::Time(0))
+      {
+        publishTwist(0, 0);
```

Comment:
I think it's better to publish the last velocity received from `cmd_vel_input` which is usually published by autonomous controller.